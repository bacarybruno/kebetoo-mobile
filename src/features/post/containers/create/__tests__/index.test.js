import setupTest from '@app/config/jest-setup'

import CreatePost from '../index'

const givenCreatePost = setupTest(CreatePost)({
  navigation: {
    navigate: jest.fn(),
    setOptions: jest.fn(),
  },
})

it('renders CreatePost', () => {
  const { wrapper } = givenCreatePost()
  expect(wrapper.toJSON()).toMatchSnapshot()
})
