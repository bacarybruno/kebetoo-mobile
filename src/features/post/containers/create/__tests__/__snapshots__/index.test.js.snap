// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CreatePost 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
        "marginBottom": 24,
        "marginHorizontal": 16,
      },
      false,
    ]
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#F5F5F5",
        "borderColor": "transparent",
        "borderRadius": 15,
        "borderWidth": 1,
        "height": 381.14285714285717,
        "marginTop": 8,
        "minHeight": 200,
        "overflow": "hidden",
        "paddingRight": 16,
        "paddingVertical": 6,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
          "paddingLeft": 16,
        }
      }
    >
      <TextInput
        allowFontScaling={true}
        autoFocus={true}
        blurOnSubmit={false}
        defaultValue="hello world"
        maxLength={180}
        multiline={true}
        numberOfLines={8}
        onBlur={[Function]}
        onChangeText={[Function]}
        onFocus={[Function]}
        placeholder="What's on your mind?"
        placeholderTextColor="rgba(174, 174, 178, 255)"
        rejectResponderTermination={true}
        returnKeyType="done"
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "color": "rgba(0,0,0,0.87)",
              "flex": 1,
              "fontFamily": "System",
              "fontSize": 16,
              "fontWeight": "400",
              "height": "100%",
              "letterSpacing": -0.32,
              "lineHeight": 21,
            },
            Object {
              "fontSize": 16,
              "textAlignVertical": "top",
            },
            undefined,
          ]
        }
        underlineColorAndroid="transparent"
        value={null}
      />
    </View>
  </View>
  <Text
    style={
      Array [
        false,
        false,
        false,
        Object {
          "backgroundColor": "transparent",
          "color": "#000000",
          "fontFamily": "System",
          "fontSize": 12,
          "fontWeight": "400",
          "letterSpacing": 0,
          "lineHeight": 16,
        },
        Array [
          Object {
            "color": "rgba(0,0,0,0.54)",
          },
          Object {
            "backgroundColor": "transparent",
            "fontFamily": "System",
            "fontWeight": "400",
          },
          Object {
            "marginTop": 8,
          },
        ],
      ]
    }
  >
    169 characters
  </Text>
  <View
    style={
      Object {
        "flex": 1,
        "justifyContent": "flex-end",
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "flex-end",
        "flexDirection": "row",
        "justifyContent": "space-between",
        "marginTop": 16,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
          "flexDirection": "row",
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
          }
        }
      >
        <View
          accessible={true}
          focusable={true}
          hitSlop={
            Object {
              "bottom": 20,
              "left": 20,
              "right": 20,
              "top": 20,
            }
          }
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#F5F5F5",
              "borderRadius": 20,
              "height": 40,
              "justifyContent": "center",
              "marginRight": 16,
              "opacity": 1,
              "transform": Array [
                Object {
                  "scale": 1,
                },
              ],
              "width": 40,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Object {
                "color": "#0B1B77",
                "fontFamily": "icomoon",
                "fontSize": 18.018018018018015,
                "fontStyle": "normal",
                "fontWeight": "normal",
              }
            }
            text="kebeticon-camera"
          />
        </View>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
          }
        }
      >
        <View
          accessible={true}
          focusable={true}
          hitSlop={
            Object {
              "bottom": 20,
              "left": 20,
              "right": 20,
              "top": 20,
            }
          }
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#F5F5F5",
              "borderRadius": 20,
              "height": 40,
              "justifyContent": "center",
              "marginRight": 16,
              "opacity": 1,
              "transform": Array [
                Object {
                  "scale": 1,
                },
              ],
              "width": 40,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Object {
                "color": "#0B1B77",
                "fontFamily": "icomoon",
                "fontSize": 18.018018018018015,
                "fontStyle": "normal",
                "fontWeight": "normal",
              }
            }
            text="kebeticon-photo"
          />
        </View>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
          }
        }
      >
        <View
          accessible={true}
          focusable={true}
          hitSlop={
            Object {
              "bottom": 20,
              "left": 20,
              "right": 20,
              "top": 20,
            }
          }
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#F5F5F5",
              "borderRadius": 20,
              "height": 40,
              "justifyContent": "center",
              "marginRight": 16,
              "opacity": 1,
              "transform": Array [
                Object {
                  "scale": 1,
                },
              ],
              "width": 40,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Object {
                "color": "#0B1B77",
                "fontFamily": "icomoon",
                "fontSize": 18.018018018018015,
                "fontStyle": "normal",
                "fontWeight": "normal",
              }
            }
            text="kebeticon-more-h"
          />
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
        }
      }
    >
      <View
        accessible={true}
        focusable={false}
        hitSlop={
          Object {
            "bottom": 20,
            "left": 20,
            "right": 20,
            "top": 20,
          }
        }
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#F5F5F5",
            "borderRadius": 20,
            "height": 40,
            "justifyContent": "center",
            "marginRight": 16,
            "opacity": 1,
            "transform": Array [
              Object {
                "scale": 1,
              },
            ],
            "width": 40,
          }
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Object {
              "color": "#0B1B77",
              "fontFamily": "icomoon",
              "fontSize": 18.018018018018015,
              "fontStyle": "normal",
              "fontWeight": "normal",
            }
          }
          text="kebeticon-microphone"
        />
      </View>
    </View>
  </View>
</View>
`;
